### DISCUSSION TITLE
Is this project dead? Are there active alternatives?

### DISCUSSION THREAD

**Discussion by yardenac** [tremc/tremc#117] [General]
Even [stig ](https://github.com/rndusr/stig) appears abandoned. What are people using these days?

**Comment by gangelop** [comment]
It's not dead, just very deeply asleep.

**Comment by MatanZ** [comment]
You can meanwhile use my branch which is a bit more actively maintained:
https://github.com/MatanZ/tremc
And by use, I also include reporting issues and sending pull requests.

More:
transmission development stalled a bit in the last couple of years, so it is not surprising that transmission front-ends also do not see much development.

@gangelop There is now "discussion" feature in github. You should enable this, as I believe that this like a few of the other issues are more discussions than actual issues.

**Comment by gangelop** [comment]
I would consider the project alive if there were people who would be willing to regularly review pull requests. I have not been willing and able to do this for the last couple of years. 2020 especially was impossible for me. I might be able to participate again now.

**Comment by MatanZ** [comment]
So, is there any plan to improve the situation?

Transmission is back in active development again. There are plans to deprecate some RPC calls, which will leave some commands of tremc broken. There are also new features which tremc should support.

**Comment by rubictus** [comment]
Thank you for the last update (2025/07/02). I like tremc ! Much better than stig.
